tests/test_main.py

```
@@ -263,472 +201,258 @@ async def test_get_ammeter_scale_factor(fixture, expected, request):
     assert status == expected
     await charger.ws_disconnect()
 
 
 @pytest.mark.parametrize(
-    "fixture, expected", [("test_charger", None), ("test_charger_v2", None)]
+    "charger, expected",
+    [("test_charger", None), ("test_charger_v2", None)],
+    indirect=["charger"],
 )
-async def test_get_ir_temperature(fixture, expected, request):
+async def test_get_ir_temperature(charger, expected):
```

Ruff (0.14.14)

[warning] Unused function argument: `expected`

(ARG001)